src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -28,6 +28,17 @@ export class CalciteTipManager {
   //  Properties
   //
   // --------------------------------------------------------------------------
+  /**
+   * Alternate text for closing the Tip Manager.
+   */
+  @Prop({ reflect: true }) closed = false;
+
+  @Watch("closed")
+  closedChangeHandler() {
+    this.direction = null;
+    this.calciteTipManagerClosedChange.emit();
```

Comment:
Event added so that someone could know if opened/closed.

Reply:
If the intent of this rename is to indicate a toggling of state - maybe drop the Closed part.